speculos/mcu/ocr.py

```diff
@@ -81,14 +84,51 @@ def display_char(font: bagl_font.Font, char: str) -> None:
     char = get_char(font, char)
     print("\n".join(split_bytes(char.bitmap, font.bpp * char.char.char_width)))
 
+def get_json_font(json_name:str) -> Mapping[Char, BitMapChar]:
+    # If no json filename was provided, just return
+    if json_name is None:
+        return None
+
+    # Build the filename by adding api level information
+    json_name += f"-api-level-{OCR.api_level}.json"
+    # Now add the fonts path (JSON files are in speculos/fonts)
+    script_path = os.path.abspath(os.path.dirname(__file__))
+    json_name = os.path.join(script_path, "..", "fonts", json_name)
```

Comment:
You need to access the ressource following the cx_lib model: https://github.com/LedgerHQ/speculos/blob/master/speculos/main.py#L152